### PR TITLE
Commerce 7310

### PR DESCRIPTION
**Pull request by christian-chiappa**

### CODE REVIEW

**Comment by christian-chiappa** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ExportCommerceOrderReportMVCResourceCommand.java [hunk 1]
```diff
@@ -73,84 +79,134 @@ protected void doServeResource(
...
+				commerceOrder.getCommerceCurrency(),
+				commerceOrder.getTaxAmount(), themeDisplay.getLocale())
+		).put(
+			"totalWithTax", commerceOrder.getTotalWithTaxAmountMoney()
+		).put(
+			"purchaseOrderNumber", commerceOrder.getPurchaseOrderNumber()
+		).put(
+			"locale", themeDisplay.getLocale()
+		);
+
```
Comment (ivicac) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ExportCommerceOrderReportMVCResourceCommand.java:205]: @christian-chiappa can you please sort these values alphabetically?

Comment (christian-chiappa) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ExportCommerceOrderReportMVCResourceCommand.java]: @ivicac sorted

**Comment by ivicac** [comment]
ci:test:format

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**